checks.d/vsphere.py

```diff
@@ -429,12 +430,23 @@ def _get_server_instance(self, instance):
             'vcenter_host:{0}'.format(instance.get('host')),
         ]
 
+        ssl_verify = instance.get('ssl_verify', True)
+        ssl_capath = instance.get('ssl_capath', None)
+        if not ssl_verify:
+            context = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
+            context.verify_mode = ssl.CERT_NONE
+        elif ssl_capath:
+            context = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
+            context.verify_mode = ssl.CERT_REQUIRED
+            context.load_verify_locations(capath=ssl_capath)
+
         if i_key not in self.server_instances:
             try:
                 server_instance = connect.SmartConnect(
-                    host=instance.get('host'),
-                    user=instance.get('username'),
-                    pwd=instance.get('password')
+                    host = instance.get('host'),
+                    user = instance.get('username'),
+                    pwd = instance.get('password'),
+                    sslContext = context if not ssl_verify or ssl_capath else None
```

Comment:
If the customer accidentally sets `ssl_verify` to False and also sets an `ssl_capath` in the config, the `not ssl_verify` context takes precedence (which I think makes sense) but we should probably log it somewhere to help identify that case come support-time.